hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMRatisSnapshots.java

```diff
@@ -184,6 +203,7 @@ public void shutdown() {
 
   @ParameterizedTest
   @ValueSource(ints = {100})
+  @Disabled
```

Comment:
please remove these disabled annotations and try running the 100 ci tests so all the tests are included.  I want to know if this new test has any effect when all are run.